test/demo/ingress.yaml

```diff
@@ -0,0 +1,19 @@
+# SPDX-FileCopyrightText: The RamenDR authors
+# SPDX-License-Identifier: Apache-2.0
+---
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: demo-ingress
+spec:
+  rules:
+  - host:
+    http:
+      paths:
+      - path: /demo
```

Comment:
/demo forward the request `GET /demo` to nginx, but we did not configure it to serve
anything at this location, so the request will fail. Without configuration the server we can use only `/`.

This change works for me:

```
--- a/test/demo/ingress.yaml
+++ b/test/demo/ingress.yaml
@@ -7,10 +7,9 @@ metadata:
   name: demo-ingress
 spec:
   rules:
-  - host:
-    http:
+  - http:
       paths:
-      - path: /demo
+      - path: /
         pathType: Prefix
         backend:
           service:
```